src/hats/inspection/visualize_catalog.py

```diff
@@ -49,19 +50,42 @@ def _read_point_map(catalog_base_dir):
     return file_io.read_fits_image(map_file_pointer)
 
 
-def plot_points(catalog: Catalog, plot_title: str | None = None, **kwargs):
-    """Create a visual map of the input points of an in-memory catalog.
-
+def plot_density(catalog: Catalog, *, plot_title: str | None = None, order=None, unit=None, **kwargs):
+    """Create a visual map of the density of input points of a catalog on-disk.
     Args:
         catalog (`hats.catalog.Catalog`) Catalog to display
         plot_title (str): Optional title for the plot
+        order (int): Optionally reduce the display healpix order, and aggregate smaller tiles.
         kwargs: Additional args to pass to `plot_healpix_map`
     """
-    if not catalog.on_disk:
+    if catalog is None or not catalog.on_disk:
         raise ValueError("on disk catalog required for point-wise visualization")
     point_map = _read_point_map(catalog.catalog_base_dir)
-    default_title = f"Catalog point density map - {catalog.catalog_name}"
-    return plot_healpix_map(point_map, title=default_title if plot_title is None else plot_title, **kwargs)
+    map_order = hp.npix2order(len(point_map))
+
+    if order is not None:
+        if order > map_order:
+            raise ValueError(f"plotting order should be less than stored density map order ({map_order})")
+        ## Create larger pixel sums from the constituent pixels.
+        point_map = point_map.reshape((hp.order2npix(order), 2 ** (2 * (map_order - order)))).sum(axis=1)
```

Comment:
Better readability, but doesn't check the size of `point_map` array:
```suggestion
        point_map = point_map.reshape(hp.order2npix(order), -1).sum(axis=1)
```